core/libretro/vmem_utils.cpp

```diff
@@ -198,7 +198,12 @@ void vmem_platform_reset_mem(void *ptr, unsigned size_bytes) {
 	// Mark them as non accessible.
 	mprotect(ptr, size_bytes, PROT_NONE);
 	// Tell the kernel to flush'em all (FIXME: perhaps unmap+mmap 'd be better?)
+	#if defined(_SYS_MMAN_H)
```

Comment:
I doubt `_SYS_MMAN_H` is specific to Haiku and and it's not an "official" flag of any sort.

Why not use `__HAIKU__` here?

Reply:
My reasoning was that other POSIX OSes could benefit from this. If it's too specific and more harm than good, we can for sure replace with `__HAIKU__`.